packages/angular_devkit/architect/BUILD.bazel

```diff
@@ -37,20 +32,27 @@ ts_json_schema(
     src = "src/progress-schema.json",
 )
 
+ts_json_schema(
+    name = "operator_schema",
+    src = "builders/operator-schema.json",
+)
+
 ts_library(
     name = "architect",
     package_name = "@angular-devkit/architect",
     srcs = glob(
-        include = ["src/**/*.ts"],
+        include = [
+            "src/**/*.ts",
+            "builders/*.ts",
+        ],
         exclude = ["**/*_spec.ts"],
     ) + [
-        # @external_begin
```

Comment:
Guessing we to keep the `@external_begin & end` here as well to not break google3 when this is synced in

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051